_stack-threats/STA-37.md

```diff
@@ -5,9 +5,15 @@ ID: STA-37
 Threat: Power Glitching
 ThreatDescription: Power glitching microprocessors are designed to operate from a stable voltage wherein interruptions of the power supply are likely to crash running applications or reset the circuit. A power glitch will affect both the stored and the threshold values. Different internal capacities will cause the values to be influenced differently, possibly resulting in a misinterpretation of the actual value.
 ThreatOrigin: A Review of Smartcard Security Issues [^212]
-ExploitExample:
-CVEExample:
-PossibleCountermeasures: {}
+ExploitExample: Implementing Practical Electrical Glitching Attacks[^309]
+
+CVEExample:CVE-2020-35553
```

Comment:
The formatting for CVEExample and PossibleCountermeasures is incorrect. Please see other threats for formatting examples.